ee/server/services/ddp-streamer/Server.ts

```diff
@@ -67,18 +85,34 @@ export class Server extends EventEmitter {
 	async subscribe(client: Client, packet: IPacket): Promise<void> {
 		try {
 			if (!this._subscriptions.has(packet.name)) {
-				throw new Error(`Subscription '${packet.name}' doesn't exist`);
+				// TODO should not be MethorError
```

Comment:
Why not?

Reply:
I think the name is misleading.. this is a *subscription* , so using a *MethodError* seems wrong.. 

In my local changes I've removed both error types and kept just a single one called `MeteorError`, to match the `errorType` property.. but I was not sure about that as well.